security-framework-sys/src/passwords.rs

```diff
@@ -0,0 +1,34 @@
+use core_foundation_sys::base::{OSStatus, CFTypeRef};
+use libc::{c_char, c_void};
+
+use base::{SecKeychainItemRef, SecKeychainRef};
+
+extern "C" {
+    #[cfg(target_os = "macos")]
+    pub fn SecKeychainFindGenericPassword(keychainkeychainOrArray: CFTypeRef,
```

Comment:
These should be placed in the module corresponding to the header in which they are defined - `keychain` in this case I believe.